docs/test-tools/test-backend-service.md

```diff
@@ -1,8 +1,10 @@
 # Test deployed service
 
-1. Testing with manually generated Azure Active Directory Token
+1. Testing manually with an Azure Active Directory Token
```

Comment:
Lets not say manually here. We can say 

Testing Backend Apis directly with generated Azure Active Directory Token